### PR TITLE
[Makefile] added new db-fixtures command.

### PR DESCRIPTION
**Pull request by hhamon**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,6 +32,9 @@ db-create:      ## Create the database and load the fixtures in it
 	$(CONSOLE) doctrine:schema:create
 	$(CONSOLE) doctrine:fixtures:load -n
 
+db-fixtures:    ## Reloads the data fixtures for the dev environment
+	$(CONSOLE) doctrine:fixtures:load
```
Comment (tgalopin) [inline Makefile]: `-n` ?

Comment (hhamon) [inline Makefile]: true that! fixed! thanks.

File: Makefile [hunk 2]
```diff
@@ -32,6 +32,9 @@ db-create:      ## Create the database and load the fixtures in it
 	$(CONSOLE) doctrine:schema:create
 	$(CONSOLE) doctrine:fixtures:load -n
 
+db-fixtures:    ## Reloads the data fixtures for the dev environment
```
Comment (HeahDude) [inline Makefile]: Please add it to the `install` task then.

Comment (HeahDude) [inline Makefile]: Ah ok you didn't remove it from db-create, so you should :)

**Pull request merged**